tests/testthat/test-bs-theme-preset-bootswatch.R

```diff
@@ -0,0 +1,22 @@
+test_that("theme_version() is up to date", {
+  bs_versions <- dir(
+    system.file("lib", package = "bslib"),
+    pattern = "bs\\d+"
+  )
+  bs_versions <- sub("bs", "", bs_versions)
+
+  LATEST_SUPPORTED <- 5
+  bs_next <- LATEST_SUPPORTED + 1
+
+  if (bs_next %in% bs_versions) {
+    expect_equal(
+      try(theme_version(bs_theme(!!bs_next)), silent = TRUE),
+      !!as.character(bs_next),
+      label = "Bootstrap 6 was added, please update theme_version()."
+    )
+  }
```

Comment:
This will still pass if `$bootstrap-version` gets updated (and `theme_version()` doesn't), right? In that case, I'd probably prefer that we keep the current `theme_version()` implementation (and drop this test).

Reply:
Sounds good, I'll go back to the olds version